external_search.py

```diff
@@ -372,7 +425,7 @@ def base_index_name(self, index_or_alias):
         return base_works_index
 
     def create_search_doc(self, query_string, filter, pagination,
-                          debug, return_raw_results):
```

Comment:
This argument no longer makes sense since searches must now always return `Hit` objects (so they can be turned into `WorkSearchResult` objects).